Added CUAD dataset (#2219)
huggingface/datasets: datasets/cuad/README.md

---
YAML tags:

> - copy-paste the tags obtained with the tagging app: https://github.com/huggingface/datasets-tagging

Author: Just started with that!

Contract review costs also affect consumers. Since contract review costs are so prohibitive, contract review is not often performed outside corporate transactions. Small companies and individuals consequently often sign contracts without even reading them, which can result in predatory behavior that harms consumers. Automating contract review by openly releasing high-quality data and fine-tuned models can increase access to legal support for small businesses and individuals, so that legal support is not exclusively available to wealthy companies.

> To reduce the disparate societal costs of contract review, and to study how well NLP models generalize to specialized domains, we introduce a new large-scale dataset for contract review. As part of The Atticus Project, a non-profit organization of legal experts, we introduce CUAD, the Contract Understanding Atticus Dataset. This dataset was created with a year-long effort pushed forward by dozens of law student annotators, lawyers, and machine learning researchers. The dataset includes more than 500 contracts and more than 13,000 expert annotations that span 41 label categories. For each of 41 different labels, models must learn to highlight the portions of a contract most salient to that label. This makes the task a matter of finding needles in a haystack.

Reviewer: ```suggestion
To reduce the disparate societal costs of contract review, and to study how well NLP models generalize to specialized domains, the authors introduce a new large-scale dataset for contract review. As part of The Atticus Project, a non-profit organization of legal experts, CUAD is introduced, the Contract Understanding Atticus Dataset. This dataset was created with a year-long effort pushed forward by dozens of law student annotators, lawyers, and machine learning researchers. The dataset includes more than 500 contracts and more than 13,000 expert annotations that span 41 label categories. For each of 41 different labels, models must learn to highlight the portions of a contract most salient to that label. This makes the task a matter of finding needles in a haystack.
```
Maybe we can try to avoid the use of "we" everywhere to refer to the authors, since we=HF community

Author: Few touchups were in to-do for the README. Thanks!

### Source Data

#### Initial Data Collection and Normalization

> The CUAD includes commercial contracts selected from 25 different types of contracts based on the contract names as shown below. Within each type, we randomly selected contracts based on the names of the filing companies across the alphabet.

Reviewer: ```suggestion
The CUAD includes commercial contracts selected from 25 different types of contracts based on the contract names as shown below. Within each type, the creators randomly selected contracts based on the names of the filing companies across the alphabet.
```

	Consulting Agreement:		11
	Development Agreement:		29
	Distributor Agreement:		32
	Endorsement Agreement:		24
	Franchise Agreement:		15
	Hosting Agreement:		20
	IP Agreement:			17
	Joint Venture Agreemen:		23
	License Agreement:		33
	Maintenance Agreement:		34
	Manufacturing Agreement:	17
	Marketing Agreement:		17
	Non-Compete/No-Solicit/Non-Disparagement Agreement: 3
	Outsourcing Agreement:		18
	Promotion Agreement:		12
	Reseller Agreement:		12
	Service Agreement:		28
	Sponsorship Agreement:		31
	Supply Agreement:		18
	Strategic Alliance Agreement:	32
	Transportation Agreement:	13
	TOTAL:				510

#### Who are the source language producers?

The contracts were sourced from EDGAR, the Electronic Data Gathering, Analysis, and Retrieval system used at the U.S. Securities and Exchange Commission (SEC). Publicly traded companies in the United States are required to file certain contracts under the SEC rules. Access to these contracts is available to the public for free at https://www.sec.gov/edgar. Please read the Datasheet at https://www.atticusprojectai.org/ for information on the intended use and limitations of the CUAD.

### Annotations

#### Annotation process

> Our labeling process included multiple steps to ensure accuracy:

Reviewer: ```suggestion
The labeling process included multiple steps to ensure accuracy:
```

4. Category-by-Category Report Review: law students exported the labeled clauses into reports, review each clause category-by-category and highlight clauses that they believe are mislabeled.
5. Attorney Review: experienced attorneys reviewed the category-by-category report with students comments, provided comments and addressed student questions. When applicable, attorneys discussed such results with the students and reached consensus. Students made changes in eBrevia accordingly.
6. eBrevia Extras Review. Attorneys and students used eBrevia to generate a list of “extras”, which are clauses that eBrevia AI tool identified as responsive to a category but not labeled by human annotators. Attorneys and students reviewed all of the “extras” and added the correct ones. The process is repeated until all or substantially all of the “extras” are incorrect labels.
7. Final Report: The final report was exported into a CSV file. Volunteers manually added the “Yes/No” answer column to categories that do not contain an answer.

#### Who are the annotators?

Answered in above section.

### Personal and Sensitive Information

> Some clauses in the files are redacted because the party submitting these contracts redacted them to protect confidentiality. Such redaction may show up as asterisks (***) or underscores (___) or blank spaces. The dataset and the answers reflect such redactions. For example, the answer for “January __ 2020” would be “1/[]/2020”).

Reviewer: ```suggestion
Some clauses in the files are redacted because the party submitting these contracts redacted them to protect confidentiality. Such redaction may show up as asterisks (\*\*\*) or underscores (\_\_\_) or blank spaces. The dataset and the answers reflect such redactions. For example, the answer for “January \_\_ 2020” would be “1/[]/2020”).
```
escape stars and underscores

---
annotations_creators:
- expert-generated
language_creators:
- found
languages:
- en
licenses:
- cc-by-4.0
multilinguality:
- monolingual
size_categories:
- 10K<n<100K
source_datasets:
- original
task_categories:
- question-answering
task_ids:

> - closed-domain-qa

Reviewer: ```suggestion
- closed-domain-qa
- extractive-qa
```
